src/Host/Client/Impl/Host/RHost.cs

```diff
@@ -230,6 +245,56 @@ public sealed partial class RHost : IDisposable, IRExpressionEvaluator {
             await RespondAsync(request, ct, input);
         }
 
+        public Task<SendBlobResult> SendBlobAsync(byte[] data, CancellationToken ct) {
```

Comment:
These 3 methods are almost identical.
The only difference is the type of request and corresponding result. 
The same thing then propagated to RSession.

From the protocol point of view - what's the difference between blob message and evaluation message?